src/frontend/hooks/server/deviceInfo.ts

```diff
@@ -21,10 +24,33 @@ export const useEditDeviceInfo = () => {
   return useMutation({
     mutationKey: ['device'],
     mutationFn: async (name: string) => {
-      return mapeoApi.setDeviceInfo({name});
+      return mapeoApi.setDeviceInfo({
+        name,
+        deviceType: deviceType ? expoToCoreDeviceType(deviceType) : undefined,
+      });
     },
     onSuccess: () => {
       queryClient.invalidateQueries({queryKey: [DEVICE_INFO_KEY]});
     },
   });
 };
+
+function expoToCoreDeviceType(d: DeviceType): DeviceInfo['deviceType'] {
+  switch (d) {
```

Comment:
i ordered these based on likelihood given that the app probably only supports phones and tablets. gotta get that 0.0000000001% speed boost 😎